src/src/WebServer/NotificationPage.cpp

```diff
@@ -142,7 +142,19 @@ void handle_notifications() {
         html_TD();
         addHtmlInt(NotificationSettings.Port);
       }
-      else {
+      else
+      {
+          //MFD: we display the GPIO
+          if (NotificationSettings.Pin1>0){
```

Comment:
The default pin config is -1 (or at least it should be)
So better check for `>= 0` or else you cannot connect a buzzer to GPIO-0 (or at least not get it to display this)

Reply:
ok. thanks.